internal/state.go

```diff
@@ -49,32 +47,28 @@ func PrepareEngineState(ctx context.Context, engine string, sessionID string, ap
 			}
 		}
 	}()
-	if sessionID != "" {
-		doc, err = global.GetActiveDoc(ctx)
-		if doc != nil {
-			appID := "SessionDoc"
-			LogVerbose("Document: " + appID + "(reconnected)")
+	doc, err = global.GetActiveDoc(ctx)
+	if doc != nil {
+		// There is an already opened doc!
+		if appID != "" {
+			LogVerbose("App: " + appID + "(reconnected)")
 		} else {
-			LogVerbose("No active doc: " + appID + ", session=" + sessionID)
+			LogVerbose("Session app (reconnected)")
```

Comment:
We should probably add the `sessionID` to the log print here, so it's clear which session it reconnects to.

Reply:
The session id is already part of an earlier log event, so no need to add it here.